### ISSUE TITLE
Allow dots and dashes for function arguments

### ISSUE THREAD

**Issue by karovski** [eternime/superscript#4]
Per title. Ex: ^getDate(11.05.2015) will not get omitted during function/tags parsing.

**Comment by silentrob** [comment]
Yes this is a little weak, right now the function params are all type string. I can see this being improved to support boolean, integer and arrays too. There might be a library that will do most of the heavy lifting.
As an aside. the message object already parses the object looking for dates. https://github.com/eternime/superscript/blob/master/lib/message.js#L295-L329
